card/mod/05_standard/set/self/signin.rb

```diff
@@ -96,6 +95,10 @@ def default_core_args args={}
   end  
 end
 
+event :signin_success, :after=>:signin do
+  abort :success
+end
+
```

Comment:
The signin event always aborted the action so it wasn't possible to do something after a signin. I moved the `abort :success` to a separate event.